src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1072,17 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString commitHash = GIT_COMMIT_HASH != QString() ? QStringLiteral(GIT_COMMIT_HASH) : QUESTION_MARK;
+    QString branch = GIT_BRANCH != QString() ? QStringLiteral(GIT_BRANCH) : QUESTION_MARK;
+    return QStringLiteral("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n")
+        .append(QStringLiteral("<tr><td colspan=\"2\" style=\"font-style:italic;\">%1<br></td></tr>").arg(tr("It would be useful to copy and include information below when reporting an issue!")))
+        .append(QStringLiteral("<tr><td style=\"font-weight: 800\">%1:</td><td>%2: %3</td></tr>").arg(tr("Build information"), tr("Build"), mudlet::self()->version))
+        .append(QStringLiteral("<tr><td></td><td>%1: %2</td></tr>").arg(tr("OS"), QSysInfo::prettyProductName()))
```

Comment:
**I18n:** sorry but you are going to have to include the `: ` (colon AND space) inside the translatable string as some Far Eastern logographic languages (Chinese/Japanese/Korean) will use a different character (a *full-width* colon here and *may* not include a space)...
```suggestion
        .append(QStringLiteral("<tr><td></td><td>%1</td></tr>").arg(tr("OS: %1").arg(QSysInfo::prettyProductName())))
```

Reply:
Corrected.

Reply:
Is it ok like that or better like you've suggested?